site/source/docs/getting_started/FAQ.rst

```diff
@@ -103,6 +103,22 @@ Why do I get ``error while loading shared libraries: libtinfo.so.5``?
 
 LLVM and clang link libtinfo dynamically. On some recent Linuxes you may have only ``libtinfo.so.6`` (while our builders target the last Ubuntu LTS). To fix this, you can do something like ``apt-get install libtinfo5`` on Debian or Ubuntu, or on Fedora something like ``dnf install ncurses-compat-libs``.
 
+Why do I get ``machine type must be wasm32`` errors during linking?
+===================================================================
+
+That error means the linker was given files that contain something other than
+wasm object code. You can run the ``file`` command-line utility to see what
```

Comment:
Technically this error message means you gave the linker and IR of the wrong type.   The error would look different if you gave it a non-object-file, or and ELF file for example.